src/index.html

```diff
@@ -5,8 +5,320 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="logo_img">
+        <img src="https://github.com/mate-academy/layout_moyo-header/blob/master/src/images/logo.png?raw=true" alt="Logo">
```

Comment:
Don't use link for the src. You have all images in the images folder 
![image](https://github.com/mate-academy/layout_catalog/assets/99403198/a4beabcf-9dd2-4561-85ff-e6f16285c6f3)